### DISCUSSION TITLE
只有在飞行模式下才能体验满血版的苹果，怎么解决

### DISCUSSION THREAD

**Discussion by mikeq95** [NSRingo/.github#12] [Q&A 问答] [answered]
在飞行模式➕全局代理可以正常打开news,可当我用Wi-Fi或者流量时候它就打不开了，也无法体验到满血功能，怎么解决？![image](https://github.com/user-attachments/assets/14a1b038-f03d-4dda-a346-3ece96d4b31a)

**Comment by VirgilClyne** [comment, marked as answer]
https://nsringo.github.io/guide/apple-news.html#%E7%AC%AC-6-%E6%AD%A5

**Reply by mikeq95** [reply]
> https://nsringo.github.io/guide/apple-news.html#%E7%AC%AC-6-%E6%AD%A5

那么iPad能用吗，目前iPados18.1美版

**Reply by VirgilClyne** [reply]
没有写明版本限制的就是通用
